addons/sale_product_configurator/static/tests/tours/product_configurator_advanced_ui.js

```diff
@@ -78,30 +78,6 @@ tour.register('sale_product_configurator_advanced_tour', {
     trigger: '.main_product strong:contains("Custom, White, PAV9, PAV5, PAV1")',
     extra_trigger: '.oe_optional_products_modal',
     run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("Custom: Custom 1")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("PAV9: Custom 2")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("PAV9: Custom 3")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("PA5: PAV1")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("PA7: PAV1")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
-}, {
-    trigger: '.main_product div:contains("PA8: PAV1")',
-    extra_trigger: '.oe_optional_products_modal',
-    run: function () {} //check
```

Comment:
These checks are no longer required, as the modal's main product description no longer contains the custom attributes previously configured. Those are carried over to the values in the newly added variant configuration options in the modal. The extra test makes sure those values are correctly carried over.